### PR TITLE
2.13.X DDF-3915 Update ddf and solr scripts for Windows

### PR DESCRIPTION
**Pull request by mcalcote**
#### What does this PR do?
Update the DDF Windows scripts to pull extra Solr properties when starting Solr. 

#### Who is reviewing it? 
@pklinef @oconnormi @rzwiefel @ahoffer @ethantmanns @kcover @bakejeyner @tyler30clemens @figliold 

#### How should this be tested?
Inspect changes, run start scripts

#### Any background context you want to provide?
These should "just work" for the default developer use case, so please check that your normal routine is unaffected. 

#### What are the relevant tickets?
[DDF-3915](https://codice.atlassian.net/browse/DDF-3915)

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/ddf-common/src/main/resources/bin/ddfsolr.bat [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	REM Not used. From Solr 7.4 docs:
+	REM    Enable either SOLR_SSL_NEED_CLIENT_AUTH or SOLR_SSL_WANT_CLIENT_AUTH but not both at
+	REM    the same time. They are mutually exclusive and Jetty will select one of them which
+	REM    may not be what you expect.
+	SET SOLR_SSL_WANT_CLIENT_AUTH=false
+)
+
+CALL %SOLR_EXEC% %COMMAND% -p %solr.http.port%
+
+IF "%solr.http.protocol%"=="http" ECHO **** USING UNSECURED SOLR CONFIGURATION ****
```
Comment (figliold) [inline distribution/ddf-common/src/main/resources/bin/ddfsolr.bat]: ❓ Should be say: "USING INSECURE SOLR CONFIGURATION"?

File: distribution/ddf-common/src/main/resources/bin/ddf.bat [hunk 2]
```diff
@@ -4,55 +4,44 @@ SETLOCAL
...
-  DEL "%DIRNAME%\restart.jvm"
-)
+IF EXIST "%DIRNAME%\restart.jvm" DEL "%DIRNAME%\restart.jvm"
 
-REM Get Solr port to run on
-for /f "tokens=2 delims==" %%G in ('findstr /i "^\w*solr.http.port=" %DDF_HOME%\etc\system.properties') do (
-    SET SOLR_PORT=%%G
-)
+REM Get Solr client property
+FOR /f "tokens=2 delims==" %%G in ('findstr /i "^\w*solr.client=" %DDF_HOME%\etc\system.properties') DO SET SOLR_CLIENT=%%G
```
Comment (figliold) [inline distribution/ddf-common/src/main/resources/bin/ddf.bat]: ❓ Can you use the get_property.bat script here?

File: distribution/ddf-common/src/main/resources/bin/ddf.bat [hunk 3]
```diff
@@ -4,55 +4,44 @@ SETLOCAL
...
 
-IF "%SOLR_PORT%" == "" (
-    ECHO Property solr.http.port not found in system.properties. Exiting.
-    EXIT /B
-)
+REM Get Solr start property
+FOR /f "tokens=2 delims==" %%G in ('findstr /i "^\w*start.solr=" %DDF_HOME%\etc\system.properties') DO SET SOLR_START=%%G
```
Comment (figliold) [inline distribution/ddf-common/src/main/resources/bin/ddf.bat]: ❓ Can you use the get_property.bat script here?

File: distribution/ddf-common/src/main/resources/bin/ddf.bat [hunk 4]
```diff
@@ -4,55 +4,44 @@ SETLOCAL
...
 
-REM Get Solr client property
-for /f "tokens=2 delims==" %%G in ('findstr /i "^\w*solr.client=" %DDF_HOME%\etc\system.properties') do (
-    SET SOLR_CLIENT=%%G
-)
+REM Get Karaf start property
+FOR /f "tokens=2 delims==" %%G in ('findstr /i "^\w*start.ddf=" %DDF_HOME%\etc\system.properties') DO SET KARAF_START=%%G
```
Comment (figliold) [inline distribution/ddf-common/src/main/resources/bin/ddf.bat]: ❓ Can you use the get_property.bat script here?

File: distribution/ddf-common/src/main/resources/bin/ddf.bat (hunk 2 above)
Comment (mcalcote) [reply distribution/ddf-common/src/main/resources/bin/ddf.bat]: Yes 😄 I was trying to go through all that trouble for nothing 🤦‍♂️

File: distribution/ddf-common/src/main/resources/bin/ddf.bat (hunk 3 above)
Comment (mcalcote) [reply distribution/ddf-common/src/main/resources/bin/ddf.bat]: yup

File: distribution/ddf-common/src/main/resources/bin/ddf.bat (hunk 4 above)
Comment (mcalcote) [reply distribution/ddf-common/src/main/resources/bin/ddf.bat]: 👌

**Comment by mcalcote** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by ethantmanns** [comment]
I built and verified that Solr is now starting on Windows

**Pull request merged**